man/md2man-all.sh

```diff
@@ -9,7 +9,7 @@ cd "$(dirname "$(readlink -f "$BASH_SOURCE")")"
 	pwd
 }
 
-if ! ( which go-md2man &>/dev/null ); then
+if ! ( command -v go-md2man &>/dev/null ); then
```

Comment:
1. I do not understand the need for a sub-shell here.

2. Maybe just use `type` and do not redirect stderr to /dev/null? This was the reason for the initial confusion :)

So, something like
```shell
if ! type go-md2man >/dev/null; then
...
```

Reply:
ping @cpuguy83

Reply:
Updated to not use a subshell.